### PR TITLE
Added Sentiment Analysis using Deep Learning 

### LINKED ISSUE #121: Add Sentiment Analysis using Deep Learning
### Description

I want to add the feature of Sentiment Analysis using Deep Learning with better accuracy. I will try to build a better neural network that tries to do sentiment analysis.
Please assign it to me.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Garvit414**
# Fixes Issue🛠️

Closes #121 

# Description👨‍💻 

Dependencies: Tensorflow_hub

I have implemented Sentiment analysis using deep learning and a pre-trained model which has delivered impressive results.

Target: rating
# Type of change📄

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How this has been tested✅
I have run the code on Google Colab and it gives fantastic results.

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷
Screenshot 2023-07-06 234227

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1658,3 +1660,43 @@ def objective(trial):
...
+    labels = df['rating'].tolist()
+    labels = [1 if i>=7 else 0 if i>=5 else -1 for i in labels]
+    labels = np.array(pd.get_dummies(labels), dtype=int)[:] 
+
+    return labels, text
+    
+#Split into train and test dataset    
+tmp_labels, tmp_text = load_dataset('datasets\drugsComTrain_raw.tsv', 568454)
+test_labels, test_text = load_dataset('datasets\drugsComTest_raw.tsv',500000)
+
```
Comment (Rakesh9100) [inline main.py]: Why to load and split the dataset again? This process is already done before @Garvit414

Comment (Garvit414) [reply main.py]: Actually I have to split dataset into Text and labels. So firstly, I have make function for that  and after that I have call it for splitting.

Comment (Rakesh9100) [reply main.py]: Ohkk fine.

**Comment by Rakesh9100** [comment]
Pls run `black main.py` on your editor terminal to pass the Linting workflow and then commit the new indentation changes @Garvit414

**Comment by Garvit414** [comment]
Which file I have to run main.py??

**Comment by Rakesh9100** [comment]
Yes, black lint the file main.py as this is only the python file available @Garvit414

**Comment by Garvit414** [comment]
I have black lint the file main.py. Please check @Rakesh9100

**Comment by Rakesh9100** [comment]
Lint workflow has been failed, kindly black lint for your whole file changes and check again @Garvit414

**Comment by Garvit414** [comment]
I have created #141 PR. I hope that this time all the conditions get satisfied. Please check @Rakesh9100
